projects/extension/src/containers/Popup.tsx

```diff
@@ -73,16 +73,12 @@ const Popup: FunctionComponent = () => {
   }, [])
 
   useEffect(() => {
-    let unsubscribe = () => {}
-
     if (!bg) return
     disconnectTab.current = bg.uiInterface.disconnectTab
-    unsubscribe = bg.uiInterface.onChainsChanged(() => refresh())
+    const unsubscribe = bg.uiInterface.onChainsChanged(() => refresh())
     refresh()
 
-    return () => {
-      unsubscribe && unsubscribe()
-    }
+    return unsubscribe && unsubscribe()
```

Comment:
```suggestion
    return unsubscribe
```

It can never be null, can it?

Reply:
It cannot

Reply:
Plus, you don't cant to call `unsubscribe` now, you just want to return it from `useEffect` as a clean up function